x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/feature_tour/README.md

```diff
@@ -1,44 +0,0 @@
-# Feature Tour on the Rule Management Page
```

Comment:
This tour is not related to guided onboarding. Instead, we use it occasionally to highlight new features released on the rule management page. @stephmilovic, Could you please revert the deletion of this component?

Reply:
re-added! Thanks @xcrzx for explaining over zoom :)